script/generate-mime.rb

```diff
@@ -82,7 +82,8 @@ def get_matches(parent)
   end
   if !exts.empty?
     exts.each{|x|
-      extensions[x] = type if !extensions.include?(x)
+      extensions[x] = [] if extensions[x].nil?
```

Comment:
I prefer key? and include? over [x].nil?. In general this is better since the value could also be nil. Doesn't matter here.